plugins/pubIds/doi/locale/en_US/locale.po

```diff
@@ -184,3 +184,6 @@ msgstr "File: {$title}"
 
 msgid "plugins.pubIds.doi.editor.preview.objects"
 msgstr "Item"
+
+msgid "plugins.pubIds.doi.manager.monographDois"
```

Comment:
where possible use the same `msgid` across every application. For example, `plugins.pubIds.doi.manager.submimssionDois`.